content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,242 @@
+report.
+
+This is useful because :guilabel:`Reference` displays the warehouse operation associated to the
+:guilabel:`Product`. Use the :guilabel:`From` and :guilabel:`To` location and associated
+:guilabel:`Source` documents to trace the product path and identify the type of operation.
+
+.. tip::
+   It is also possible to view the stock moves of a product directly from the product page. Open the
+   desired product page by navigating to :menuselection:`Inventory --> Products --> Products` and
+   select the product. From there, click on the :guilabel:`Product Moves` smart button, then the
+   product drop-down. This displays the stock moves history of the product.
+
+.. _inventory/putaway/stock_moves:
+
+.. example::
+   A putaway rule was purposely set up incorrectly on `Apple`, so the product was moved from
+   `WH/Stock` to `WH/Stock/Shelf 1` instead of the expected location, `WH/Stock/Fruits`. To trace
+   the path the apple moved and identify the cause of the wrong stock move, open the
+   :guilabel:`Product Moves` report in :menuselection:`Inventory app --> Reporting --> Product
+   Moves --> Apple drop-down menu` displays a list of stock moves of the apple. Click on a line to
+   open a report that is useful to trace the path the product moved and identify the cause of the
+   wrong stock move.
+
+   The :guilabel:`To` field displays where the product currently is, `WH/Stock/Shelf 1`, which is
+   the wrong location. The :guilabel:`From` location is the vendor's location, which is expected
+   since the :guilabel:`Source` of the stock move is a :abbr:`PO (Purchase Order)`.
+
+   The :guilabel:`Reference` operation, `WH/IN/00015`, which is a warehouse reception of products
+   from the vendor, would further explain why the apples were moved to the wrong location.
+
+   .. image:: putaway/stock-move-report.png
+      :align: center
+      :alt: Open stock move report
+
+To follow an operation identified from the :guilabel:`Product Moves` report, navigate to
+:menuselection:`Inventory --> Operations --> Transfers`. This is the page that displays all
+warehouse transfers-- internal or external. Search for and click on the corresponding
+:guilabel:`Reference` operation to view the document explaining why the product was moved to the
+wrong location.
+
+.. tip::
+   When there are too many records, use either the :guilabel:`Filters` or :guilabel:`Group by`
+   features to sort and :ref:`find desired records<inventory/putaway/use-advanced-filters>`. It is
+   useful to search exclusively for the receipt operation type because putaway rules store products
+   *into* a specific stock location.
+
+.. example::
+   Continuing to diagnose why the product, `Apple`, was placed in `WH/Stock/Shelf 1`, navigate to
+   the list of all transfers in :menuselection:`Inventory --> Operations --> Transfers`. Click on
+   the list item where the :guilabel:`Reference` field matches the `WH/IN/00015` operation
```

Comment:
In my suggestion I am referring to \`WH/Stock/Shelf 1\` and \`WH/IN/00015\` to have a :guilabel: but on a second look, I think it could go either way. 👍